Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/encryptionmigration/EncryptedPreferencesMigration.kt

```diff
@@ -30,7 +30,7 @@ class EncryptedPreferencesMigration @Inject constructor(
 
     private val coroutineScope: CoroutineScope = CoroutineScope(Dispatchers.IO)
 
-    fun doMigration() {
+    suspend fun doMigration() {
```

Comment:
can we use `coroutineScope` here ? and avoid making `doMigration()` suspend and therefore you don't need to change how we call it from `CoronaWarnApplication` ?

Reply:
Used `coroutineScope` and reset change in `CoronaWarnApplication`